apps/src/templates/CodeWorkspace.jsx

```diff
@@ -1,50 +1,27 @@
 var Radium = require('radium');
 var ProtectedStatefulDiv = require('./ProtectedStatefulDiv');
 var JsDebugger = require('./JsDebugger');
+var PaneHeader = require('./PaneHeader');
+var PaneSection = PaneHeader.PaneSection;
+var PaneButton = PaneHeader.PaneButton;
 var msg = require('../locale');
 var commonStyles = require('../commonStyles');
 var styleConstants = require('../styleConstants');
 var color = require('../color');
 var experiments = require('../experiments');
 
+var BLOCKS_GLYPH_LIGHT = "data:image/gif;base64,R0lGODlhEAAQAIAAAP///////yH+GkNyZWF0ZWQgd2l0aCBHSU1QIG9uIGEgTWFjACH5BAEKAAEALAAAAAAQABAAAAIdjI+py40AowRp2molznBzB3LTIWpGGZEoda7gCxYAOw==";
+var BLOCKS_GLYPH_DARK = "data:image/gif;base64,R0lGODlhEAAQAIAAAE1XX01XXyH+GkNyZWF0ZWQgd2l0aCBHSU1QIG9uIGEgTWFjACH5BAEKAAEALAAAAAAQABAAAAIdjI+py40AowRp2molznBzB3LTIWpGGZEoda7gCxYAOw==";
```

Comment:
Replaced the 256x256 image with a base64 encoded 16x16 image so that we can easily swap between light and dark without having to do an async image load (another option would be to background load in advance). We are losing something, as the original image had slightly rounded rects, but it was being rendered small enough that I don't think I can see the difference.